<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>buy</class>
 <widget class="QWidget" name="buy">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>750</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QStackedWidget" name="stack">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>750</width>
       <height>600</height>
      </size>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="query_page">
      <widget class="QLabel" name="label">
       <property name="geometry">
        <rect>
         <x>37</x>
         <y>10</y>
         <width>131</width>
         <height>41</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">font: 290 12pt &quot;微软雅黑 Light&quot;;</string>
       </property>
       <property name="text">
        <string>当前热播：</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_2">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>73</y>
         <width>141</width>
         <height>41</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">font: 290 12pt &quot;微软雅黑 Light&quot;;</string>
       </property>
       <property name="text">
        <string>观影日期：</string>
       </property>
      </widget>
      <widget class="QPushButton" name="query">
       <property name="geometry">
        <rect>
         <x>604</x>
         <y>77</y>
         <width>91</width>
         <height>41</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">font: 290 12pt &quot;微软雅黑 Light&quot;;</string>
       </property>
       <property name="text">
        <string>查询</string>
       </property>
      </widget>
      <widget class="QCheckBox" name="checkBox_1">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>85</y>
         <width>61</width>
         <height>18</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QLabel" name="label_5">
       <property name="geometry">
        <rect>
         <x>351</x>
         <y>79</y>
         <width>101</width>
         <height>31</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">font: 290 12pt &quot;微软雅黑 Light&quot;;</string>
       </property>
       <property name="text">
        <string>时间段：</string>
       </property>
      </widget>
      <widget class="QCheckBox" name="checkBox_2">
       <property name="geometry">
        <rect>
         <x>331</x>
         <y>88</y>
         <width>61</width>
         <height>18</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QComboBox" name="comboBox_1">
       <property name="geometry">
        <rect>
         <x>157</x>
         <y>17</y>
         <width>161</width>
         <height>31</height>
        </rect>
       </property>
      </widget>
      <widget class="QComboBox" name="comboBox_3">
       <property name="geometry">
        <rect>
         <x>443</x>
         <y>81</y>
         <width>91</width>
         <height>31</height>
        </rect>
       </property>
      </widget>
      <widget class="QComboBox" name="comboBox_2">
       <property name="geometry">
        <rect>
         <x>158</x>
         <y>79</y>
         <width>91</width>
         <height>31</height>
        </rect>
       </property>
      </widget>
      <widget class="QPushButton" name="select">
       <property name="geometry">
        <rect>
         <x>100</x>
         <y>164</y>
         <width>561</width>
         <height>51</height>
        </rect>
       </property>
       <property name="text">
        <string>###</string>
       </property>
      </widget>
      <widget class="QPushButton" name="select_3">
       <property name="geometry">
        <rect>
         <x>100</x>
         <y>227</y>
         <width>561</width>
         <height>51</height>
        </rect>
       </property>
       <property name="text">
        <string>###</string>
       </property>
      </widget>
      <widget class="QPushButton" name="select_4">
       <property name="geometry">
        <rect>
         <x>100</x>
         <y>291</y>
         <width>561</width>
         <height>51</height>
        </rect>
       </property>
       <property name="text">
        <string>###</string>
       </property>
      </widget>
      <widget class="QPushButton" name="select_5">
       <property name="geometry">
        <rect>
         <x>100</x>
         <y>355</y>
         <width>561</width>
         <height>51</height>
        </rect>
       </property>
       <property name="text">
        <string>###</string>
       </property>
      </widget>
      <widget class="QPushButton" name="select_6">
       <property name="geometry">
        <rect>
         <x>100</x>
         <y>417</y>
         <width>561</width>
         <height>51</height>
        </rect>
       </property>
       <property name="text">
        <string>###</string>
       </property>
      </widget>
      <widget class="QPushButton" name="select_7">
       <property name="geometry">
        <rect>
         <x>100</x>
         <y>480</y>
         <width>561</width>
         <height>51</height>
        </rect>
       </property>
       <property name="text">
        <string>###</string>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="select_page">
      <widget class="QLabel" name="l2_7">
       <property name="geometry">
        <rect>
         <x>28</x>
         <y>9</y>
         <width>151</width>
         <height>51</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">font: 25 16pt &quot;微软雅黑 Light&quot;;</string>
       </property>
       <property name="text">
        <string>选座界面：</string>
       </property>
      </widget>
      <widget class="QPushButton" name="lock">
       <property name="geometry">
        <rect>
         <x>557</x>
         <y>15</y>
         <width>101</width>
         <height>41</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">font: 25 12pt &quot;微软雅黑 Light&quot;;</string>
       </property>
       <property name="text">
        <string>锁座</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_3">
       <property name="geometry">
        <rect>
         <x>275</x>
         <y>17</y>
         <width>91</width>
         <height>41</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">font: 290 12pt &quot;微软雅黑 Light&quot;;</string>
       </property>
       <property name="text">
        <string>购票数：</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="number">
       <property name="geometry">
        <rect>
         <x>370</x>
         <y>24</y>
         <width>61</width>
         <height>31</height>
        </rect>
       </property>
      </widget>
      <widget class="QWidget" name="verticalLayoutWidget">
       <property name="geometry">
        <rect>
         <x>60</x>
         <y>80</y>
         <width>623</width>
         <height>397</height>
        </rect>
       </property>
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="4" column="3">
         <widget class="QCheckBox" name="s13">
          <property name="styleSheet">
           <string notr="true">QCheckBox::indicator {
width: 87px;
height: 60px;
color: rgb(0, 0, 0);
}
QCheckBox::indicator:unchecked {
image: url(:/icon/image/seat_1.png);
}
QCheckBox::indicator:checked {
image: url(:/icon/image/seat.png)
}
</string>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item row="3" column="1">
         <widget class="QCheckBox" name="s25">
          <property name="styleSheet">
           <string notr="true">QCheckBox::indicator {
width: 87px;
height: 60px;
color: rgb(0, 0, 0);
}
QCheckBox::indicator:unchecked {
image: url(:/icon/image/seat_1.png);
}
QCheckBox::indicator:checked {
image: url(:/icon/image/seat.png)
}
</string>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item row="4" column="2">
         <widget class="QCheckBox" name="s14">
          <property name="styleSheet">
           <string notr="true">QCheckBox::indicator {
width: 87px;
height: 60px;
color: rgb(0, 0, 0);
}
QCheckBox::indicator:unchecked {
image: url(:/icon/image/seat_1.png);
}
QCheckBox::indicator:checked {
image: url(:/icon/image/seat.png)
}
</string>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QCheckBox" name="s35">
          <property name="styleSheet">
           <string notr="true">QCheckBox::indicator {
width: 87px;
height: 60px;
color: rgb(0, 0, 0);
}
QCheckBox::indicator:unchecked {
image: url(:/icon/image/seat_1.png);
}
QCheckBox::indicator:checked {
image: url(:/icon/image/seat.png)
}
</string>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item row="4" column="5">
         <widget class="QCheckBox" name="s11">
          <property name="styleSheet">
           <string notr="true">QCheckBox::indicator {
width: 87px;
height: 60px;
color: rgb(0, 0, 0);
}
QCheckBox::indicator:unchecked {
image: url(:/icon/image/seat_1.png);
}
QCheckBox::indicator:checked {
image: url(:/icon/image/seat.png)
}
</string>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item row="2" column="5">
         <widget class="QCheckBox" name="s31">
          <property name="styleSheet">
           <string notr="true">QCheckBox::indicator {
width: 87px;
height: 60px;
color: rgb(0, 0, 0);
}
QCheckBox::indicator:unchecked {
image: url(:/icon/image/seat_1.png);
}
QCheckBox::indicator:checked {
image: url(:/icon/image/seat.png)
}
</string>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item row="2" column="3">
         <widget class="QCheckBox" name="s33">
          <property name="styleSheet">
           <string notr="true">QCheckBox::indicator {
width: 87px;
height: 60px;
color: rgb(0, 0, 0);
}
QCheckBox::indicator:unchecked {
image: url(:/icon/image/seat_1.png);
}
QCheckBox::indicator:checked {
image: url(:/icon/image/seat.png)
}
</string>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item row="3" column="0">
         <widget class="QCheckBox" name="s26">
          <property name="styleSheet">
           <string notr="true">QCheckBox::indicator {
width: 87px;
height: 60px;
color: rgb(0, 0, 0);
}
QCheckBox::indicator:unchecked {
image: url(:/icon/image/seat_1.png);
}
QCheckBox::indicator:checked {
image: url(:/icon/image/seat.png)
}
</string>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item row="1" column="3">
         <widget class="QCheckBox" name="s43">
          <property name="styleSheet">
           <string notr="true">QCheckBox::indicator {
width: 87px;
height: 60px;
color: rgb(0, 0, 0);
}
QCheckBox::indicator:unchecked {
image: url(:/icon/image/seat_1.png);
}
QCheckBox::indicator:checked {
image: url(:/icon/image/seat.png)
}
</string>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item row="3" column="5">
         <widget class="QCheckBox" name="s21">
          <property name="styleSheet">
           <string notr="true">QCheckBox::indicator {
width: 87px;
height: 60px;
color: rgb(0, 0, 0);
}
QCheckBox::indicator:unchecked {
image: url(:/icon/image/seat_1.png);
}
QCheckBox::indicator:checked {
image: url(:/icon/image/seat.png)
}
</string>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QCheckBox" name="s46">
          <property name="styleSheet">
           <string notr="true">QCheckBox::indicator {
width: 87px;
height: 60px;
color: rgb(0, 0, 0);
}
QCheckBox::indicator:unchecked {
image: url(:/icon/image/seat_1.png);
}
QCheckBox::indicator:checked {
image: url(:/icon/image/seat.png)
}
</string>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item row="3" column="4">
         <widget class="QCheckBox" name="s22">
          <property name="styleSheet">
           <string notr="true">QCheckBox::indicator {
width: 87px;
height: 60px;
color: rgb(0, 0, 0);
}
QCheckBox::indicator:unchecked {
image: url(:/icon/image/seat_1.png);
}
QCheckBox::indicator:checked {
image: url(:/icon/image/seat.png)
}
</string>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item row="3" column="2">
         <widget class="QCheckBox" name="s24">
          <property name="styleSheet">
           <string notr="true">QCheckBox::indicator {
width: 87px;
height: 60px;
color: rgb(0, 0, 0);
}
QCheckBox::indicator:unchecked {
image: url(:/icon/image/seat_1.png);
}
QCheckBox::indicator:checked {
image: url(:/icon/image/seat.png)
}
</string>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item row="4" column="1">
         <widget class="QCheckBox" name="s15">
          <property name="styleSheet">
           <string notr="true">QCheckBox::indicator {
width: 87px;
height: 60px;
color: rgb(0, 0, 0);
}
QCheckBox::indicator:unchecked {
image: url(:/icon/image/seat_1.png);
}
QCheckBox::indicator:checked {
image: url(:/icon/image/seat.png)
}
</string>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item row="0" column="3">
         <widget class="QCheckBox" name="s53">
          <property name="styleSheet">
           <string notr="true">QCheckBox::indicator {
width: 87px;
height: 60px;
color: rgb(0, 0, 0);
}
QCheckBox::indicator:unchecked {
image: url(:/icon/image/seat_1.png);
}
QCheckBox::indicator:checked {
image: url(:/icon/image/seat.png)
}
</string>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item row="0" column="5">
         <widget class="QCheckBox" name="s51">
          <property name="styleSheet">
           <string notr="true">QCheckBox::indicator {
width: 87px;
height: 60px;
color: rgb(0, 0, 0);
}
QCheckBox::indicator:unchecked {
image: url(:/icon/image/seat_1.png);
}
QCheckBox::indicator:checked {
image: url(:/icon/image/seat.png)
}
</string>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item row="0" column="4">
         <widget class="QCheckBox" name="s52">
          <property name="styleSheet">
           <string notr="true">QCheckBox::indicator {
width: 87px;
height: 60px;
color: rgb(0, 0, 0);
}
QCheckBox::indicator:unchecked {
image: url(:/icon/image/seat_1.png);
}
QCheckBox::indicator:checked {
image: url(:/icon/image/seat.png)
}
</string>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item row="2" column="2">
         <widget class="QCheckBox" name="s34">
          <property name="styleSheet">
           <string notr="true">QCheckBox::indicator {
width: 87px;
height: 60px;
color: rgb(0, 0, 0);
}
QCheckBox::indicator:unchecked {
image: url(:/icon/image/seat_1.png);
}
QCheckBox::indicator:checked {
image: url(:/icon/image/seat.png)
}
</string>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item row="1" column="4">
         <widget class="QCheckBox" name="s42">
          <property name="styleSheet">
           <string notr="true">QCheckBox::indicator {
width: 87px;
height: 60px;
color: rgb(0, 0, 0);
}
QCheckBox::indicator:unchecked {
image: url(:/icon/image/seat_1.png);
}
QCheckBox::indicator:checked {
image: url(:/icon/image/seat.png)
}
</string>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item row="1" column="5">
         <widget class="QCheckBox" name="s41">
          <property name="styleSheet">
           <string notr="true">QCheckBox::indicator {
width: 87px;
height: 60px;
color: rgb(0, 0, 0);
}
QCheckBox::indicator:unchecked {
image: url(:/icon/image/seat_1.png);
}
QCheckBox::indicator:checked {
image: url(:/icon/image/seat.png)
}
</string>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item row="3" column="3">
         <widget class="QCheckBox" name="s23">
          <property name="styleSheet">
           <string notr="true">QCheckBox::indicator {
width: 87px;
height: 60px;
color: rgb(0, 0, 0);
}
QCheckBox::indicator:unchecked {
image: url(:/icon/image/seat_1.png);
}
QCheckBox::indicator:checked {
image: url(:/icon/image/seat.png)
}
</string>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item row="2" column="4">
         <widget class="QCheckBox" name="s32">
          <property name="styleSheet">
           <string notr="true">QCheckBox::indicator {
width: 87px;
height: 60px;
color: rgb(0, 0, 0);
}
QCheckBox::indicator:unchecked {
image: url(:/icon/image/seat_1.png);
}
QCheckBox::indicator:checked {
image: url(:/icon/image/seat.png)
}
</string>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item row="1" column="2">
         <widget class="QCheckBox" name="s44">
          <property name="styleSheet">
           <string notr="true">QCheckBox::indicator {
width: 87px;
height: 60px;
color: rgb(0, 0, 0);
}
QCheckBox::indicator:unchecked {
image: url(:/icon/image/seat_1.png);
}
QCheckBox::indicator:checked {
image: url(:/icon/image/seat.png)
}
</string>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item row="4" column="4">
         <widget class="QCheckBox" name="s12">
          <property name="styleSheet">
           <string notr="true">QCheckBox::indicator {
width: 87px;
height: 60px;
color: rgb(0, 0, 0);
}
QCheckBox::indicator:unchecked {
image: url(:/icon/image/seat_1.png);
}
QCheckBox::indicator:checked {
image: url(:/icon/image/seat.png)
}
</string>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QCheckBox" name="s36">
          <property name="styleSheet">
           <string notr="true">QCheckBox::indicator {
width: 87px;
height: 60px;
color: rgb(0, 0, 0);
}
QCheckBox::indicator:unchecked {
image: url(:/icon/image/seat_1.png);
}
QCheckBox::indicator:checked {
image: url(:/icon/image/seat.png)
}
</string>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item row="0" column="2">
         <widget class="QCheckBox" name="s54">
          <property name="styleSheet">
           <string notr="true">QCheckBox::indicator {
width: 87px;
height: 60px;
color: rgb(0, 0, 0);
}
QCheckBox::indicator:unchecked {
image: url(:/icon/image/seat_1.png);
}
QCheckBox::indicator:checked {
image: url(:/icon/image/seat.png)
}
</string>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QCheckBox" name="s45">
          <property name="styleSheet">
           <string notr="true">QCheckBox::indicator {
width: 87px;
height: 60px;
color: rgb(0, 0, 0);
}
QCheckBox::indicator:unchecked {
image: url(:/icon/image/seat_1.png);
}
QCheckBox::indicator:checked {
image: url(:/icon/image/seat.png)
}
</string>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item row="4" column="0">
         <widget class="QCheckBox" name="s16">
          <property name="styleSheet">
           <string notr="true">QCheckBox::indicator {
width: 87px;
height: 60px;
color: rgb(0, 0, 0);
}
QCheckBox::indicator:unchecked {
image: url(:/icon/image/seat_1.png);
}
QCheckBox::indicator:checked {
image: url(:/icon/image/seat.png)
}
</string>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QCheckBox" name="s55">
          <property name="styleSheet">
           <string notr="true">QCheckBox::indicator {
width: 87px;
height: 60px;
color: rgb(0, 0, 0);
}
QCheckBox::indicator:unchecked {
image: url(:/icon/image/seat_1.png);
}
QCheckBox::indicator:checked {
image: url(:/icon/image/seat.png)
}
</string>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="QCheckBox" name="s56">
          <property name="styleSheet">
           <string notr="true">QCheckBox::indicator {
width: 87px;
height: 60px;
color: rgb(0, 0, 0);
}
QCheckBox::indicator:unchecked {
image: url(:/icon/image/seat_1.png);
}
QCheckBox::indicator:checked {
image: url(:/icon/image/seat.png)
}
</string>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QLabel" name="l2_10">
       <property name="geometry">
        <rect>
         <x>280</x>
         <y>510</y>
         <width>151</width>
         <height>41</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">font: 25 18pt &quot;微软雅黑 Light&quot;;</string>
       </property>
       <property name="text">
        <string>屏幕在此</string>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="pay_page">
      <widget class="QPushButton" name="pay">
       <property name="geometry">
        <rect>
         <x>189</x>
         <y>28</y>
         <width>101</width>
         <height>41</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">font: 25 10pt &quot;微软雅黑 Light&quot;;</string>
       </property>
       <property name="text">
        <string>支付接口</string>
       </property>
      </widget>
      <widget class="QLabel" name="l2_8">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>20</y>
         <width>151</width>
         <height>51</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">font: 25 16pt &quot;微软雅黑 Light&quot;;</string>
       </property>
       <property name="text">
        <string>进行支付：</string>
       </property>
      </widget>
      <widget class="QLabel" name="l2_9">
       <property name="geometry">
        <rect>
         <x>31</x>
         <y>87</y>
         <width>161</width>
         <height>41</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">font: 25 16pt &quot;微软雅黑 Light&quot;;</string>
       </property>
       <property name="text">
        <string>打印票据：</string>
       </property>
      </widget>
      <widget class="QTextBrowser" name="textBrowser_4">
       <property name="geometry">
        <rect>
         <x>50</x>
         <y>150</y>
         <width>651</width>
         <height>271</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">color:black;font-family: \&quot;simsun\&quot;; font-size:25px; font-weight:bold</string>
       </property>
       <property name="html">
        <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;meta charset=&quot;utf-8&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
hr { height: 1px; border-width: 0; }
li.unchecked::marker { content: &quot;\2610&quot;; }
li.checked::marker { content: &quot;\2612&quot;; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'&amp;quot;simsun&amp;quot;'; font-size:25px; font-weight:700; font-style:normal;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
      </widget>
      <widget class="QPushButton" name="back">
       <property name="geometry">
        <rect>
         <x>483</x>
         <y>460</y>
         <width>101</width>
         <height>41</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">font: 25 12pt &quot;微软雅黑 Light&quot;;</string>
       </property>
       <property name="text">
        <string>返回</string>
       </property>
      </widget>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
